contracts/starknet/space.cairo

```diff
@@ -278,17 +283,18 @@ func constructor{syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_check_p
     # Sanity checks
     with_attr error_message("Invalid constructor parameters"):
         assert_nn(_voting_delay)
-        assert_nn(_voting_duration)
         assert_not_zero(_controller)
         assert_not_zero(_voting_strategies_len)
         assert_not_zero(_authenticators_len)
+        assert_le(_min_voting_duration, _max_voting_duration)
         assert_not_zero(_executors_len)
     end
     # TODO: maybe use uint256_signed_nn to check proposal_threshold?
 
     # Initialize the storage variables
     voting_delay.write(_voting_delay)
-    voting_duration.write(_voting_duration)
+    min_voting_duration.write(_min_voting_duration)
+    max_voting_duration.write(_max_voting_duration)
```

Comment:
Would be nice to add event here with all these params if its easy to do or can be done in another PR

Reply:
was thinking about doing this in another PR (to close https://github.com/snapshot-labs/sx-core/issues/91) :)